pkg/network/ebpf/c/protocols/http2/decoding.h

```diff
@@ -117,9 +117,47 @@ static __always_inline void parse_field_indexed(dynamic_table_index_t *dynamic_i
 
 READ_INTO_BUFFER(path, HTTP2_MAX_PATH_LEN, BLK_SIZE)
 
+// get_bucket_index returns the bucket index for the given path size.
+static __always_inline __u32 get_bucket_index(__u32 size) {
+    if (size < HTTP2_MAX_PATH_LEN) {
+        return 0;
+    }
+    __s32 bucket_idx = (size - HTTP2_MAX_PATH_LEN) / 10;
+    bucket_idx = bucket_idx > 6 ? 6 : bucket_idx;
+    bucket_idx = bucket_idx < 0 ? 0 : bucket_idx;
+    return bucket_idx;
+}
+
+// update_path_size_telemetry updates the path size telemetry.
+static __always_inline void update_path_size_telemetry(http2_telemetry_t *http2_tel, __u32 bucket_idx) {
```

Comment:
alongside with my comment to convert `http2_tel->path_size_bucket*` into an array, you can simplify this method to be a single line